setup.py

```diff
@@ -23,9 +23,9 @@
     name="p1_data_client_python",
     description="Package for P1 Data API access",
     keywords=["p1_data_client_python", "API", "data", "financial", "economic"],
-    version="0.0.1",
-    author="p1_data_client_python",
-    author_email="",
+    version="1.0.0",
```

Comment:
Since we use version, would be good to add release in git. It will be reflected on the main page of the git repo.
![Selection_153](https://user-images.githubusercontent.com/11608884/94587000-a613f400-028a-11eb-82b0-0947343b2d27.png)
You can see an example of how it can be done in https://github.com/ParticleDev/alpha_particle_backend/releases